LibCarla/cmake/ros2-native/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+cmake_minimum_required (VERSION 3.20.0)
+
+project (carla-ros2-native)
+
+set (CARLA_WORKSPACE_PATH ${CMAKE_CURRENT_SOURCE_DIR})
+set (LIBCARLA_SOURCE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../source)
+
+file (
+    GLOB
+    LIBCARLA_ROS2_SOURCES
+    ${LIBCARLA_SOURCE_PATH}/carla/ros2/publishers/*.cpp
+    ${LIBCARLA_SOURCE_PATH}/carla/ros2/subscribers/*.cpp
+    ${LIBCARLA_SOURCE_PATH}/carla/ros2/listeners/*.cpp
+    ${LIBCARLA_SOURCE_PATH}/carla/ros2/types/*.cpp
+)
+
+file (
+    GLOB
+    LIBCARLA_ROS2_HEADERS
+    ${LIBCARLA_SOURCE_PATH}/carla/ros2/publishers/*.h
+    ${LIBCARLA_SOURCE_PATH}/carla/ros2/subscribers/*.h
+    ${LIBCARLA_SOURCE_PATH}/carla/ros2/listeners/*.h
+    ${LIBCARLA_SOURCE_PATH}/carla/ros2/types/*.h
+)
+
+add_library (
+    carla-ros2-native SHARED
+    ${LIBCARLA_ROS2_HEADERS}
+    ${LIBCARLA_ROS2_SOURCES}
+)
+
+target_include_directories (
+    carla-ros2-native SYSTEM PRIVATE
+    ${LIBCARLA_SOURCE_PATH}
+    ${FASTDDS_INCLUDE_PATH}
+    ${BOOST_INCLUDE_PATH}
+)
+
+target_compile_definitions (
+    carla-ros2-native PUBLIC
+    BOOST_ASIO_ENABLE_BUFFER_DEBUGGING
+)
+
+target_link_libraries(carla-ros2-native /home/xsole/programing/carlaUE5/Install/fast-dds-install/lib/libfastrtps.so /home/xsole/programing/carlaUE5/Install/fast-dds-install/lib/libfastcdr.so)
```

Comment:
can you change these paths to be relative instead?

Reply:
Sure. I'm working on it, thank you!